### PR TITLE
Play dead upon taking damage instead of immediately upon using ability

### PR DESCRIPTION
**Pull request by katlings**

### CODE REVIEW

File: gamemode/init.lua [hunk 1]
```diff
@@ -161,7 +161,8 @@ local function BroadcastPlayerDeath(ply)
     net.Broadcast()
     -- remove ragdoll
     local ragdoll = ply:GetRagdollEntity()
-    SafeRemoveEntityDelayed(ragdoll, 5)
+    -- ideally this ragdoll stays around the same amount of time as a prop can play dead for
+    SafeRemoveEntityDelayed(ragdoll, 8)
```
Comment (Calvin-L) [inline gamemode/init.lua]: Maybe move `playdead_duration` from `player_ext.lua` to a constant in `sh_config.lua` so you can use it here?

**Comment by Calvin-L** [comment]
This all seems legit 👍

File: gamemode/init.lua (hunk 1 above)
Comment (katlings) [reply gamemode/init.lua]: lmao thanks I knew I wanted it to be a constant but I couldn't figure out where to put it at 1 in the morning

**Comment by kvovin** [comment]
Sweet

**Pull request merged**